src/_includes/side_nav.html

```diff
@@ -6,12 +6,14 @@
       {% for item in site.pages %}
         {% assign item_crumbs = item.url | split: '/' %}
         {% if item_crumbs.size == 2 and item.name == 'index.md' %}
-          <li>
-            <a class="{% if item.url == page.url %}current{% endif %}" href="{{ item.url | relative_url }}">{{ item.title }}</a>
-            {% if item.url == page.url or item.url == parent_page.url %}
+          <li class="related__section-link">
               {% assign grouped_related_pages = site.pages | children_of: item | group_by: "related_order" | sort: "name" %}
               {% if grouped_related_pages.size > 0 %}
-                <ul class="nested-navigation">
+                <a class="{% if item.url == page.url %}current{% endif %}" href="{{ item.url | relative_url }}">{{ item.title }}</a>
```

Comment:
Does this line need to be inside the `if`? I can't see what's different in the `else` version?

Reply:
yes its a hangover from an older iteration that I squashed - thanks for spotting :)